{"uid":"e12435671bd3df01","name":"test_given_vault_kv_requirer_related_when_create_secret_then_secret_is_created","fullName":"tests.integration.test_integration.TestVaultK8sIntegrationsPart1#test_given_vault_kv_requirer_related_when_create_secret_then_secret_is_created","historyId":"b73cafaa4a16ef737fd64806526cc114","time":{"start":1736952815966,"stop":1736952815966,"duration":0},"status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: vault-k8s/0","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 840, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 875, in pytest_runtest_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 697, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 532, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 617, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1047, in execute\n    fixturedef = request._get_active_fixturedef(argname)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 617, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1071, in execute\n    raise exc.with_traceback(exc_tb)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 323, in _async_fixture_wrapper\n    return event_loop.run_until_complete(setup())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 320, in setup\n    res = await func(**_add_kwargs(func, kwargs, event_loop, request))\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/tests/integration/test_integration.py\", line 78, in deployed_vault\n    await ops_test.model.wait_for_idle(\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 3203, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 3094, in _raise_for_status\n    raise error_type(\njuju.errors.JujuUnitError: Unit in error: vault-k8s/0","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1736952747646,"stop":1736952747646,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"ops_test","time":{"start":1736952747646,"stop":1736952748088,"duration":442},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"event_loop","time":{"start":1736952747646,"stop":1736952747646,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"deployed_vault","time":{"start":1736952748088,"stop":1736952815953,"duration":67865},"status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: vault-k8s/0\n","statusTrace":"  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 323, in _async_fixture_wrapper\n    return event_loop.run_until_complete(setup())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 320, in setup\n    res = await func(**_add_kwargs(func, kwargs, event_loop, request))\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/tests/integration/test_integration.py\", line 78, in deployed_vault\n    await ops_test.model.wait_for_idle(\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 3203, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/ubuntu/actions-runner/_work/vault-k8s-operator/vault-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 3094, in _raise_for_status\n    raise error_type(\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"afterStages":[{"name":"ops_test::finalizer","time":{"start":1736952815993,"stop":1736952819325,"duration":3332},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"event_loop::2","time":{"start":1736952819326,"stop":1736952819326,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"event_loop::_close_event_loop","time":{"start":1736952819326,"stop":1736952819326,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1736952819326,"stop":1736952819326,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_integration"},{"name":"subSuite","value":"TestVaultK8sIntegrationsPart1"},{"name":"host","value":"github-runner"},{"name":"thread","value":"22543-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_integration"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":3,"skipped":1,"passed":11,"unknown":0,"total":15},"items":[{"uid":"ad13cc986e484277","reportUrl":"https://canonical.github.io/vault-k8s-operator/2718//#testresult/ad13cc986e484277","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: vault-k8s/0","time":{"start":1736948961786,"stop":1736948961786,"duration":0}},{"uid":"f3a846419acc3bdb","reportUrl":"https://canonical.github.io/vault-k8s-operator/2717//#testresult/f3a846419acc3bdb","status":"passed","time":{"start":1736899111482,"stop":1736899113532,"duration":2050}},{"uid":"442bf90bd1f773d9","reportUrl":"https://canonical.github.io/vault-k8s-operator/2718//#testresult/442bf90bd1f773d9","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: vault-k8s/0","time":{"start":1736899938794,"stop":1736899938794,"duration":0}},{"uid":"3ff29c7575edc6d4","reportUrl":"https://canonical.github.io/vault-k8s-operator/2716//#testresult/3ff29c7575edc6d4","status":"passed","time":{"start":1736894710424,"stop":1736894714511,"duration":4087}},{"uid":"89affb930ef918b2","reportUrl":"https://canonical.github.io/vault-k8s-operator/2715//#testresult/89affb930ef918b2","status":"passed","time":{"start":1736892908547,"stop":1736892912630,"duration":4083}},{"uid":"c2a14ed7f4ad043c","reportUrl":"https://canonical.github.io/vault-k8s-operator/2714//#testresult/c2a14ed7f4ad043c","status":"passed","time":{"start":1736864045369,"stop":1736864047422,"duration":2053}},{"uid":"4f056b339b2645a","reportUrl":"https://canonical.github.io/vault-k8s-operator/2713//#testresult/4f056b339b2645a","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1736864924196,"stop":1736864924196,"duration":0}},{"uid":"7823adcb8165f683","reportUrl":"https://canonical.github.io/vault-k8s-operator/2711//#testresult/7823adcb8165f683","status":"passed","time":{"start":1736854356476,"stop":1736854359539,"duration":3063}},{"uid":"1c47b07fa352f37d","reportUrl":"https://canonical.github.io/vault-k8s-operator/2710//#testresult/1c47b07fa352f37d","status":"passed","time":{"start":1736807172811,"stop":1736807174870,"duration":2059}},{"uid":"34d397c40052f6f7","reportUrl":"https://canonical.github.io/vault-k8s-operator/2709//#testresult/34d397c40052f6f7","status":"passed","time":{"start":1736802980700,"stop":1736802983771,"duration":3071}},{"uid":"b4da501349c6c5df","reportUrl":"https://canonical.github.io/vault-k8s-operator/2708//#testresult/b4da501349c6c5df","status":"passed","time":{"start":1736799512174,"stop":1736799516238,"duration":4064}},{"uid":"62ed5d63d3ad0b65","reportUrl":"https://canonical.github.io/vault-k8s-operator/2707//#testresult/62ed5d63d3ad0b65","status":"passed","time":{"start":1736797603698,"stop":1736797606759,"duration":3061}},{"uid":"cfe821eab33198a3","reportUrl":"https://canonical.github.io/vault-k8s-operator/2689//#testresult/cfe821eab33198a3","status":"passed","time":{"start":1736782718058,"stop":1736782722138,"duration":4080}},{"uid":"ece66d4f5b5174e5","reportUrl":"https://canonical.github.io/vault-k8s-operator/2689//#testresult/ece66d4f5b5174e5","status":"passed","time":{"start":1736775607420,"stop":1736775611495,"duration":4075}}]},"tags":["asyncio","abort_on_fail"]},"source":"e12435671bd3df01.json","parameterValues":[]}